src/reducers/GlobalReducer.js

```diff
@@ -0,0 +1,20 @@
+const GlobalReducer = (state, action) => {
+    switch(action.type) {
+        case 'CHANGE_SEARCH_TEXT': {
+            return {
+                ...state,
+                searchText: action.payload.searchText,
+            }
+        };
+        case 'changeTheme': {
```

Comment:
Try to apply a standard for the action type names. Here we have upper and camel case types, this opens to human errors.

A good practice is to use javascript const in order to set the action types names.

```js
// actions-types.js

export const CHANGE_SEARC_TEXT = 'CHANGE_SEARC_TEXT';
// ... rest of actions types

```

for example.